lib/assets/javascripts/react_ujs_mount.js

```diff
@@ -53,7 +53,7 @@
 
         // Assume className is simple and can be found at top-level (window).
         // Fallback to eval to handle cases like 'My.React.ComponentName'.
-        var constructor = window[className] || eval.call(window, className);
+        var constructor = window[className].default || window[className] || eval.call(window, className);
```

Comment:
if `window[className]` returns undefined, will this throw `TypeError: Cannot read property 'default' of undefined` ?